packages/shared/client/ReplayClient.ts

```diff
@@ -580,7 +589,7 @@ export class ReplayClient implements ReplayClientInterface {
 
       hitCounts.set(location.line, {
         columnHits: [...previous.columnHits, columnHits],
-        hits: previous.hits + hits,
+        hits: Math.max(hits, previous.hits),
```

Comment:
was this what we were doing before?

Reply:
Functionally, I'm not sure– but I think it's what makes the most sense. If we're combining all hits on a line to a single number in the gutter, I think it makes sense to show the highest number.

Reply:
I think I see a problem on production related to this, in fact.

Looking at recording 768b3f13-0635-4dbc-8b8e-e3f53a70e405, and opening `bundle_input.js` – I see the backend return the following for `getHitCounts`:
```json
      {
        "location": {
          "sourceId": "ppo2-0-bf6b0aa22d03fb56ecb173a045d549ab9e5a9f5f0d646bc12cf33587db090dfe-34da5b56d657c786b881afc8211b9a9e4d692c26ff995edfe976f7fddced4e7d",
          "line": 1,
          "column": 15
        },
        "hits": 56
      }
```

My branch shows:
![Screen Shot 2022-10-13 at 11 11 07 PM](https://user-images.githubusercontent.com/29597/195753733-0b0ac3ec-7fe8-47ba-9f5d-a3048e482e80.png)

Production shows:
![Screen Shot 2022-10-13 at 11 11 13 PM](https://user-images.githubusercontent.com/29597/195753746-dc9b65a2-3f6e-41fe-8346-c96779ba0329.png)

Reply:
The rationale for summing the hit counts is that that's what you'll see when you add a print statement

Reply:
Thanks for explaining this. It set me off down a bit of a rabbit trail, but I learned:
* We merge hit counts for all corresponding sources
* We only count the hits for the first breakable column on any given line (for each source)
* We _have_ to _explicitly_ fetch breakable points for each source before querying their hit counts, or the backend will incorrectly say that we've asked about an "invalid location" (which I filed BAC-2370 about)

This branch and production now match in terms of the hit count labels displayed.